Student/Student/Pages/PageList/PageListCell.swift

```diff
@@ -0,0 +1,34 @@
+//
+// This file is part of Canvas.
+// Copyright (C) 2019-present  Instructure, Inc.
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as
+// published by the Free Software Foundation, either version 3 of the
+// License, or (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+//
+
+import UIKit
+import Core
+
+class PageListCell: UITableViewCell {
+    @IBOutlet weak var iconImageView: IconView!
+    @IBOutlet weak var titleLabel: DynamicLabel!
+    @IBOutlet weak var dateLabel: DynamicLabel!
+
+    func update(page: Page, color: UIColor?) {
+        let formatter = DateFormatter()
+        formatter.dateFormat = "MMM d' at 'h:mm a"
```

Comment:
I don't believe the `dateFormat` property is i18n friendly. All other languages could end up with a literal `" at "` in their date string, which won't make sense to them. Setting `dateStyle` & `timeStyle` is preferred, or you can use `setLocalizedDateFormatFromTemplate` if needed.